lib/minitest/parallel_fork/fail_fast.rb

```diff
@@ -1,69 +1,15 @@
+require_relative './halt'
 require_relative '../parallel_fork'
 
 module Minitest::ParalleForkFailFast
+  include Minitest::Halt
+
   def run_after_parallel_fork_hook(i)
```

Comment:
Seems like this should be moved to `halt.rb` as well.  It's used by both `fail_fast.rb` and `ctrl_c.rb`, so it would avoid duplication of code.